inmoose/cohort_qc/cohort_metric.py

```diff
@@ -0,0 +1,864 @@
+# -----------------------------------------------------------------------------
+# Copyright (C) 2022-2023 M. Colange
```

Comment:
credit the actual author(s) of the file with the proper date
probably 2024 L. Meunier